ArduPlane/is_flying.cpp

```diff
@@ -23,8 +23,16 @@ void Plane::update_is_flying_5Hz(void)
                                     (gps.ground_speed_cm() >= ground_speed_thresh_cm);
 
     // airspeed at least 75% of stall speed?
-    bool airspeed_movement = ahrs.airspeed_estimate(&aspeed) && (aspeed >= (MAX(aparm.airspeed_min,2)*0.75f));
-
+    const float airspeed_threshold = MAX(aparm.airspeed_min,2)*0.75f;
+    bool airspeed_movement = ahrs.airspeed_estimate(&aspeed) && (aspeed >= airspeed_threshold);
+
+    if (gps.status() < AP_GPS::GPS_OK_FIX_2D && arming.is_armed() && !airspeed_movement && isFlyingProbability > 0.3) {
+        // when flying with no GPS, use the last airspeed estimate to
+        // determine if we think we have airspeed movement. This
+        // prevents the crash detector from triggering when
+        // dead-reckoning under long GPS loss
+        airspeed_movement = aspeed >= airspeed_threshold;
```

Comment:
For clarity, please separate the bool statements from the assignment parts. Easier to catch mistakes that way.
`airspeed_movement = (aspeed >= airspeed_threshold);`